dev-java/core-specs-alpha/core-specs-alpha-0.1.24.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+JAVA_PKG_IUSE="test"
+
+inherit java-pkg-2 java-ant-2
```

Comment:
Can this be sorted?

Reply:
Can't due to the following `* java-ant-2 eclass can only be inherited AFTER java-pkg-2 or java-pkg-opt-2`

Reply:
ok